clab/config.go

```diff
@@ -470,6 +472,10 @@ func (c *cLab) NewLink(l LinkConfig) *Link {
 	link := new(Link)
 	link.Labels = l.Labels
 
+	if link.MTU == 0 {
+		link.MTU = 1500
+	}
```

Comment:
Better check if `link.MTU <= 0` to set the default 1500. the netlink package just converts the int passed mtu using `uint32(mtu)`. it can end up with silly values if the given MTU is negative.

BTW shouldn't MTU be added as a config option in `LinkConfig` ?

Reply:
changed the condition to `<=`
Yes, MTU will be added under LinkConfig, I didn't give it a thought on how to better do this. I think LinkConfig format needs to be discussed
For now that option stays unexported to a user...